apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -244,8 +245,16 @@ function initializeBlocklyWrapper(blocklyInstance) {
     blocklyWrapper.Block.prototype.getTitleValue;
 
   blocklyWrapper.cdoUtils = {
-    loadBlocksToWorkspace(blockSpace, xml) {
-      return Blockly.Xml.domToBlockSpace(blockSpace, xml);
+    loadBlocksToWorkspace(blockSpace, source) {
+      const isXml = stringIsXml(source);
+      if (!isXml) {
+        console.warn(
+          `Source string was JSON. Use Version History to recover a working version of this project.`,
+          `This likely occurred by opening a project that was last saved with Google Blockly.`
```

Comment:
Should we include this line about Google Blockly? I could see that being confusing to users. The first line is probably sufficient.

Reply:
My thinking was that this would only seen by someone who is using the `?blocklyVersion` flag, which means they're probably internal.

Reply:
gotcha, makes sense